runtime/src/blockhash_queue.rs

```diff
@@ -4,13 +4,28 @@ use solana_sdk::hash::Hash;
 use solana_sdk::timing::timestamp;
 use std::collections::HashMap;
 
+#[allow(dead_code)]
+const FULL_SPEND_BLOCKHASH_STR: &str = "Fu11Spend1111111111111111111111111111111111";
```

Comment:
This belongs in sdk/ with an enormous comment!

Reply:
Bytes too?

Reply:
Yeah, we do that for others.  eg: https://github.com/solana-labs/solana/blob/master/sdk/src/bpf_loader.rs

Reply:
you can also put in the test code that panics if they differ with a "please update me" message

Reply:
@rob-solana I get that test code for free if I use `solana_name_id!` macro, right?

Reply:
yeah, but solana_name_id! does pubkeys... a small investment in generalization might be nice to have